Tools/BuildAndPackage.ps1

```diff
@@ -78,7 +78,7 @@ function CleanUp-PrivateDataAndExportVariables($moduleManifest) {
 
     $moduleManifest.Remove("PrivateData")
     $moduleManifest["VariablesToExport"] = $toBeReplaced
-    $moduleManifest["FunctionsToExport"] = $toBeReplaced
+    $moduleManifest["FunctionsToExport"] = @("gs:")
```

Comment:
I removed it now. It should just read it from the file. Previously, we want it blank.